tests/test_self.py

```diff
@@ -122,11 +121,11 @@ def _display(self, layout: "Section") -> None:
         pass
 
     @property
-    def out(self) -> StringIO:
+    def out(self) -> TextIO:  # type: ignore # Property is not the same as BaseLinter's attr
         return self._reporters[0].out
 
-    @property
-    def linter(self) -> Optional[Any]:
+    @property  # type: ignore # Property is not the same as BaseLinter's attr
+    def linter(self) -> PyLinter:  # type: ignore
```

Comment:
`type: ignore[override]` should only be necessary on the second line.

Reply:
I agree, but `mypy` doesn't. Note that we also don't get a "useless suppression" warning. Might want to report this as a bug to `mypy`, but both comments are necessary for `mypy` to pass.